android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Jun 23 08:50:38 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3-bin.zip
```

Comment:
What's the difference between the all and bin version of gradle?

Reply:
Also. Can we align the versions, so they use the same version of 7.x?

Reply:
@jonasanker it seems they are independent. Looking through the gradle settings in Android Studio it seems a new version of gradle was released yesterday (7.4) 

![image](https://user-images.githubusercontent.com/39860858/153154221-1eba4ce0-7729-4f12-903b-4750fc2b822c.png)

Reply:
All good then. I'll approve